codegen/src/test/resources/software/amazon/awssdk/codegen/poet/builder/test-async-client-builder-class.java

```diff
@@ -28,12 +29,17 @@ public DefaultJsonAsyncClientBuilder tokenProvider(SdkTokenProvider tokenProvide
     }
 
     @Override
-    protected final JsonAsyncClient buildClient() {
+    protected JsonAsyncClient buildClient() {
```

Comment:
Why remove final?

Reply:
intellij suggested to remove it, added it back